developer/src/kmc-kmn/test/test-messages.ts

```diff
@@ -96,30 +96,30 @@ describe('KmnCompilerMessages', function () {
 
   // ERROR_OutsTooLong
 
-  it('should generate ERROR_OutsTooLong if a store referenced in outs() is too long (more than GLOBAL_BUFSIZE elements)', async function() {
+  it('should generate ERROR_OutsTooLong if a store referenced in outs() is too long and would overflow the buffer', async function() {
     await testForMessage(this, ['invalid-keyboards', 'error_outs_too_long.kmn'], KmnCompilerMessages.ERROR_OutsTooLong);
-    // callbacks.printMessages();
+    assert.equal(callbacks.messages[0].message, "Store cannot be inserted with outs() as it makes the extended string too long character offset: 5");
```

Comment:
I am not sure if we really want to test the message content because adjusting the text of a message should probably not be affecting unit tests?

Reply:
Indeed, but I limited myself to just correcting the four failing tests.  Six other pre-existing tests check the error message, only two do not, so for consistency I replaced commented out code to just print the message with assertions for the four tests I touched.  Options include (i) simply removing the assertions on the messages on the four corrected tests, accepting the inconsistency; (ii) removing the assertions on messages from all existing tests as well as the four I corrected; (iii) looking to find a way (not sure how difficult this would be as I am not yet fully familiar with the interaction of the TS wrapper and kmcmplib) to read the messages from kmcmplib and assert these in the TS ... not really necessary in a unit test though ... operating more at the functional level; (iv) look to mock some aspect of the interaction so there is no dependency on the actual message text in kmcmplib.

Given effort vs. return, I am thinking (ii); what do you think?

Reply:
I think (ii) makes sense, yes

Reply:
Done